frontend/src/modules/calculator/containers/Keyboard.js

```diff
@@ -0,0 +1,266 @@
+                    </CustomCell>
+                    <CustomCell>
+                        <OperationKey onClick={() => addExpression('×')}>×</OperationKey>
+                    </CustomCell>
+                </TableRow>
+                <TableRow>
+                    <CustomCell>
+                        <FunctionKey onClick={() => addExpression('e ')}>e</FunctionKey>
+                    </CustomCell>
+                    <CustomCell>
+                        {isInv ? (
+                            <FunctionKey onClick={() => addExpression('tan<sup>-1</sup>(')}>
+                                tan<CustomSup>-1</CustomSup>
+                            </FunctionKey>
+                        ) : (
+                            <FunctionKey onClick={() => addExpression('tan(')}>tan</FunctionKey>
+                        )}
+                    </CustomCell>
+                    <CustomCell>
+                        {isInv ? (
+                            <FunctionKey onClick={() => addExpression('<sup>2</sup>')}>
+                                x<CustomSup>2</CustomSup>
+                            </FunctionKey>
+                        ) : (
+                            <FunctionKey onClick={() => addExpression('√(')}>√</FunctionKey>
+                        )}
+                    </CustomCell>
+                    <CustomCell>
+                        <NumberKey onClick={() => addExpression('1')}>1</NumberKey>
+                    </CustomCell>
+                    <CustomCell>
+                        <NumberKey onClick={() => addExpression('2')}>2</NumberKey>
+                    </CustomCell>
+                    <CustomCell>
+                        <NumberKey onClick={() => addExpression('3')}>3</NumberKey>
+                    </CustomCell>
+                    <CustomCell>
+                        <OperationKey onClick={() => addExpression('-')}>-</OperationKey>
+                    </CustomCell>
+                </TableRow>
+                <TableRow>
+                    <CustomCell>
+                        {isInv ? (
+                            <FunctionKey onClick={() => addExpression(randomGenerate())}>Rnd</FunctionKey>
+                        ) : (
+                            <FunctionKey onClick={() => addExpression('Ans')}>Ans</FunctionKey>
+                        )}
+                    </CustomCell>
+                    <CustomCell>
+                        <FunctionKey
+                            onClick={() => {
+                                if (isPreviousNumber(currentExpression.join(''))) addExpression('E');
+                                else dispatch(setError('E должно стоять после числа'));
+                            }}
+                        >
+                            EXP
+                        </FunctionKey>
+                    </CustomCell>
+                    <CustomCell>
+                        {/* {isInv ? ( */}
```

Comment:
@thienlongtpct can you remove obsolete code?

Reply:
Ok, I will remove it. I want to add some other function there, but then I forgot, that is why there is still obsolete code.